akka-cluster-sharding/src/test/scala/akka/cluster/sharding/EntityTerminationSpec.scala

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import akka.actor.Actor
+import akka.actor.Props
+import akka.cluster.Cluster
+import akka.cluster.MemberStatus
+import akka.testkit.AkkaSpec
+import akka.testkit.ImplicitSender
+import akka.testkit.WithLogCapturing
+import com.typesafe.config.ConfigFactory
+
+import scala.concurrent.duration._
+
+/**
+ * Verifies that the automatic restart on terminate/crash that is in place for remember entities does not apply
+ * when remember entities is not enabled
+ */
+object EntityTerminationSpec {
+
+  final case class EntityEnvelope(id: String, msg: Any)
+
+  def config = ConfigFactory.parseString("""
+      akka.loglevel=DEBUG
+      akka.loggers = ["akka.testkit.SilenceAllTestEventListener"]
+      akka.actor.provider = cluster
+      akka.remote.artery.canonical.port = 0
+      akka.remote.classic.netty.tcp.port = 0
+      akka.cluster.sharding.state-store-mode = ddata
+      # no leaks between test runs thank you
+      akka.cluster.sharding.distributed-data.durable.keys = []
+      akka.cluster.sharding.verbose-debug-logging = on
+      akka.cluster.sharding.entity-restart-backoff = 500ms
```

Comment:
make it a bit smaller to speed up the test?

Reply:
Will do, set it high becayse CI, but let's start with a smaller value and work it upwards if it fails